test/Mono.Linker.Tests.Cases/Warnings/WarningSuppression/SuppressWarningsInMembersAndTypes.cs

```diff
@@ -103,9 +103,7 @@ public void SuppressionHasNullParameters ()
 
 	class SuppressOnTypeMarkedEntirely
 	{
-		// https://github.com/mono/linker/issues/2095
-		// [LogDoesNotContain (nameof(TypeWithSuppression) + " has more than one unconditional suppression")]
-		[LogContains (nameof (TypeWithSuppression) + " has more than one unconditional suppression")]
+		[LogDoesNotContain (nameof (TypeWithSuppression) + " has more than one unconditional suppression")]
```

Comment:
Can you please also add a test case for https://github.com/dotnet/runtime/issues/56252.
Basically a "copy" assembly which has compiler generated method with a suppression - in that case we don't guarantee order of marking between the method and compiler generated code - and that broke suppressions.